src/DripsHub.sol

```diff
@@ -828,6 +829,22 @@ abstract contract DripsHub {
         return UserOrAccount({isAccount: true, user: user, account: account});
     }
 
+    function _hashUser(UserOrAccount memory userOrAccount) internal pure returns (bytes32) {
+        if (userOrAccount.isAccount) {
+            return _hashUser(userOrAccount.user, userOrAccount.account);
+        } else {
+            return _hashUser(userOrAccount.user);
+        }
+    }
+
+    function _hashUser(address user) internal pure returns (bytes32) {
+        return keccak256(abi.encode(user));
+    }
+
+    function _hashUser(address user, uint256 account) internal pure returns (bytes32) {
+        return keccak256(abi.encode(user, account));
+    }
```

Comment:
I would make both functions public. It just easier of another contract wants to build on top and can use these functions.

Reply:
That's a good point, fixed